packages/commonwealth/client/scripts/views/pages/discussions/ThreadCard/AuthorAndPublishInfo/AuthorAndPublishInfo.tsx

```diff
@@ -24,6 +24,25 @@ import { NewThreadTag } from '../../NewThreadTag';
 import './AuthorAndPublishInfo.scss';
 import useAuthorMetadataCustomWrap from './useAuthorMetadataCustomWrap';
 
+function formatVersionText(
```

Comment:
create `utils.ts` file next to `AuthorAndPublishInfo.tsx` and move this function right there, using `const formatVersionText = () => {}` syntax